#ifndef ROBOT_MSG_HEADER
#define ROBOT_MSG_HEADER

//ros messages header
#include <gobot_msgs/AllSensorsMsg.h>
#include <gobot_msgs/AlignmentMsg.h>
#include <gobot_msgs/BatteryMsg.h>
#include <gobot_msgs/BumperMsg.h>
#include <gobot_msgs/CliffMsg.h>
#include <gobot_msgs/EncodersMsg.h>
#include <gobot_msgs/GyroMsg.h>
#include <gobot_msgs/IrMsg.h>
#include <gobot_msgs/LedMsg.h>
#include <gobot_msgs/MotorSpeedMsg.h>
#include <gobot_msgs/OdomTestMsg.h>
#include <gobot_msgs/ProximityMsg.h>
#include <gobot_msgs/SonarMsg.h>
#include <gobot_msgs/SoundMsg.h>
#include <gobot_msgs/WeightMsg.h>

//ros services header
#include <gobot_msgs/GetEncoders.h>
#include <gobot_msgs/GetGobotStatus.h>
#include <gobot_msgs/GetInt.h>
#include <gobot_msgs/GetIntArray.h>
#include <gobot_msgs/GetString.h>
#include <gobot_msgs/GetStringArray.h>
#include <gobot_msgs/InitBack.h>
#include <gobot_msgs/IsCharging.h>
#include <gobot_msgs/SendMap.h>
#include <gobot_msgs/SendMessageToPc.h>
#include <gobot_msgs/SetBattery.h>
#include <gobot_msgs/SetGobotStatus.h>
#include <gobot_msgs/SetBool.h>
#include <gobot_msgs/SetInt.h>
#include <gobot_msgs/SetIntArray.h>
#include <gobot_msgs/SetSpeeds.h>
#include <gobot_msgs/SetString.h>
#include <gobot_msgs/SetStringArray.h>
#include <gobot_msgs/SetFloatArray.h>
#endif
